xbmc/network/httprequesthandler/python/HTTPPythonWsgiInvoker.h

```diff
@@ -52,7 +52,7 @@ class CHTTPPythonWsgiInvoker : public CHTTPPythonInvoker
 
 private:
   static std::map<std::string, std::string> createCgiEnvironment(const HTTPPythonRequest* httpRequest, ADDON::AddonPtr addon);
-  static void addWsgiEnvironment(HTTPPythonRequest* request, void* environ);
+  static void addWsgiEnvironment(HTTPPythonRequest* request, void* environment);
```

Comment:
IIRC this was an issue on another platform. I first had it named `environment` as well but then the build failed on some platform but I don't remember which one.